### PR TITLE
Adapting Readmes and some more references from IDU to HL

### PR DESCRIPTION
**Pull request by Echsecutor**

### CODE REVIEW

File: test/docker-compose.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ services:
       - .env
     restart: always
     networks:
-      idunion_local:
+      local:
```
Comment (mgmgwi) [inline test/docker-compose.yaml:24]: I would like a more specific name for the network, e.g. indy-nodes-local or something like that

**Review by mgmgwi — COMMENTED** [review]
back in a few hours...

File: test/docker-compose.yaml (hunk 1 above)
Comment (Echsecutor) [reply test/docker-compose.yaml]: good point :+1:

File: build/Dockerfile.bullseye [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # This container is to run indy-node.
-# It is part of https://github.com/hyperledger/indy-node-container . See there for LICENCE, copyright, etc.
+# This file is part of https://github.com/hyperledger/indy-node-container .
+# Copyright 2021-2022 by all poeple listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```
Comment (mgmgwi) [inline build/Dockerfile.bullseye:3]:
```suggestion
# Copyright 2021-2022 by all people listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```

File: build/Dockerfile.buster [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # This container is to run indy-node.
-# It is part of https://github.com/hyperledger/indy-node-container . See there for LICENCE, copyright, etc.
+# This file is part of https://github.com/hyperledger/indy-node-container .
+# Copyright 2021-2022 by all poeple listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```
Comment (mgmgwi) [inline build/Dockerfile.buster:3]:
```suggestion
# Copyright 2021-2022 by all people listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```

File: build/Dockerfile.ubuntu16 [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # This container is to run indy-node.
-# It has been created in the indy-node docker working group of the ID Union project.
-# author: Robin Klemens <klemens@intnernet-sicherheit.de>
+# This file is part of https://github.com/hyperledger/indy-node-container .
+# Copyright 2021-2022 by all poeple listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```
Comment (mgmgwi) [inline build/Dockerfile.ubuntu16:3]:
```suggestion
# Copyright 2021-2022 by all people listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```

File: build/Dockerfile.ubuntu20 [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # This container is to run indy-node.
-# It has been created in the indy-node docker working group of the ID Union project.
+# This file is part of https://github.com/hyperledger/indy-node-container .
+# Copyright 2021-2022 by all poeple listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```
Comment (mgmgwi) [inline build/Dockerfile.ubuntu20:3]:
```suggestion
# Copyright 2021-2022 by all people listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```

File: build/init_and_run.sh [hunk 6]
```diff
@@ -2,6 +2,10 @@
 
 # This script provides idempotent initialization and finally runs the indy node inside the docker container
 
+# This file is part of https://github.com/hyperledger/indy-node-container .
+# Copyright 2021-2022 by all poeple listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```
Comment (mgmgwi) [inline build/init_and_run.sh:6]:
```suggestion
# Copyright 2021-2022 by all people listed in https://github.com/hyperledger/indy-node-container/blob/main/NOTICE , see
```

**Review by mgmgwi — APPROVED** [review]

**Pull request merged**